services/welding-machine-data-simulator-service/app/routers/simulator_router.py

```
@@ -0,0 +1,77 @@
+
+@router.post("/start")
+async def start_simulator():
+    """시뮬레이션 시작"""
+    try:
+        await simulator_scheduler.start()
+        return {
+            "message": "시뮬레이터가 시작되었습니다.",
+            "status": simulator_scheduler.get_status()
+        }
+    except Exception as e:
+        raise HTTPException(status_code=500, detail=f"시뮬레이터 시작 실패: {str(e)}")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)